maps/z2.dmm

```diff
@@ -46085,20 +46014,6 @@
 /turf/unsimulated/floor/blackwhite,
 /area/owlery/office)
 "cKv" = (
-/obj/critter/zombie/security{
-	aggressive = 0;
-	alive = 0;
-	atcritter = 0;
-	atkcarbon = 0;
-	atksilicon = 0;
-	butcherable = 0;
-	can_revive = 0;
-	density = 0;
-	health = 0;
-	icon_state = "seczombie-dead";
-	layer = 2;
-	name = "Gutted Corpse"
-	},
```

Comment:
The existing code is gross as hell but these should be replaced with something, rather than straight removed- maybe just make a subtype of `/obj/decal/fakeobjects/skeleton` which has the dead zombie icon?

Reply:
Was planning to replace them with corpse spawners at a later time